src/utils/auth.js

```diff
@@ -0,0 +1,32 @@
+const passport = require('passport');
+
+/**
+ * Helper function to obtain the token from the authorization header
+ *
+ * @param {*} req - the express request
+ * @returns the bearer token if it exists, otherwise null
+ */
+function getToken(req) {
+  const authHeader = req.headers.authorization;
+  if (authHeader) return authHeader.split(' ')[1];
```

Comment:
Might be good just as a reminder to comment that this expects the header to be of the form "Bearer {token}".